mmv1/third_party/terraform/services/alloydb/resource_alloydb_cluster_restore_test.go

```diff
@@ -93,6 +93,12 @@ resource "google_alloydb_instance" "source" {
   cluster       = google_alloydb_cluster.source.name
   instance_id   = "tf-test-alloydb-instance%{random_suffix}"
   instance_type = "PRIMARY"
+	client_connection_config {
```

Comment:
These should have the same indentation level as the previous line.

Reply:
Done